src/options.js

```diff
@@ -602,12 +616,12 @@ export function maybeNamed(things) {
   return isIterable(things) ? named(things) : things;
 }
 
-// TODO Accept other types of clips (paths, urls, x, y, other marks…)?
-// https://github.com/observablehq/plot/issues/181
 export function maybeClip(clip) {
   if (clip === true) clip = "frame";
   else if (clip === false) clip = null;
-  else if (clip != null) clip = keyword(clip, "clip", ["frame", "sphere"]);
+  else if (isGeoJSON(clip)) {
+    if (clip.type === "Sphere") clip = "sphere";
```

Comment:
note that other geojson pass through

Reply:
Let’s promote `clip = "sphere"` to `{type: "Sphere"}` here so we have one fewer code path later?